docs/google_analytics_custom_import.md

```diff
@@ -18,6 +15,32 @@
         - Name: "External Link Click Status"
         - ID: 'ga:dimension37'
     - Overwrite Hit Data: 'No'
-7. Next go to Admin -> User Management in Google Analytics, under your domain and grant the service account you setup in step 1 edit access: `https://www.googleapis.com/auth/analytics.edit`.
-8. Run the rake task `export:google_analytics:bad_links` to make your first upload to Google Analytics. It should show up in the Custom Import you set up on step 5. 
-       
+4. Next go to Admin -> User Management in Google Analytics, under your domain and grant the service account you setup in step 1 edit access: `https://www.googleapis.com/auth/analytics.edit`.
+5. The app assumes five environment variables are made available through `govuk-secrets`: 
+- `GOOGLE_CLIENT_EMAIL` (this is extracted from the JSON file containing the credentials for your service account, in step 1)
+- `GOOGLE_PRIVATE_KEY` (this is extracted from the JSON file containing the credentials for your service account, in step 1)
+- `GOOGLE_EXPORT_ACCOUNT_ID` (this is obtained from the GA account domain you are trying to upload to. We use the account id for `www.gov.uk`)
+- `GOOGLE_EXPORT_TRACKER_ID` (this is the web property UA string associated with the upload. Ex: `UA-XXXXXXXXX-X`)
+- `GOOGLE_EXPORT_CUSTOM_DATA_IMPORT_SOURCE_ID` (this is obtained from the custom data import setup in GA. Ex: `abcdeFGHijk12_AB12cd`)
+
+If these environment variables are not already set, you need to extract them and make them available to your app via `govuk-secrets`.
+
+6. In a development environment, you can test the authentication by adding your environment variables in `local_env.yml`, which should look like this:
+
+---
+GOOGLE_CLIENT_EMAIL: 'linkstatusuploader@badlinksexporter.iam.gserviceaccount.com'
+GOOGLE_PRIVATE_KEY: 'abcd'
+GOOGLE_EXPORT_ACCOUNT_ID: '1234'
+GOOGLE_EXPORT_CUSTOM_DATA_IMPORT_SOURCE_ID: 'UA-12345-2'
+GOOGLE_EXPORT_TRACKER_ID: 'abcdef'
+
```

Comment:
I think it might be more "correct" to combine 5 & 6 here.  Something like:

> 5.  The app assumes the following environment variables are set and made available to it:
>
>  [LIST OF ENV VARS + DESCRIPTIONS AS IS]
> 
> In development mode you can add these to your `local_env.yml` file and they will be turned into environment variables for you.  In deployed environments we manage these through govuk-puppet and govuk-secrets.